dart/lib/src/protocol/sentry_user.dart

```diff
@@ -63,12 +63,18 @@ class SentryUser {
 
   /// Deserializes a [SentryUser] from JSON [Map].
   factory SentryUser.fromJson(Map<String, dynamic> json) {
+    Map<String, dynamic>? extras;
+    if (json['extras'] != null) {
+      extras = Map<String, dynamic>.from(json['extras'] as Map);
+    } else {
+      extras = null;
+    }
```

Comment:
```suggestion
    var extras = json['extras'];
    if (extras != null) {
      extras = Map<String, dynamic>.from(extras as Map);
    }
```

I think that would be cleaner